contracts/modules/Experimental/TransferManager/LockupVolumeRestrictionTM.sol

```diff
@@ -61,166 +62,218 @@ contract LockupVolumeRestrictionTM is ITransferManager {
     {
     }
 
-
     /** @notice Used to verify the transfer transaction and prevent locked up tokens from being transferred
      * @param _from Address of the sender
      * @param _amount The amount of tokens to transfer
-     * @param _isTransfer Whether or not this is an actual transfer or just a test to see if the tokens would be transferrable
      */
-    function verifyTransfer(address  _from, address /* _to*/, uint256  _amount, bytes /* _data */, bool  _isTransfer) public returns(Result) {
+    function verifyTransfer(address  _from, address /* _to*/, uint256  _amount, bytes /* _data */, bool /*_isTransfer*/) public returns(Result) {
         // only attempt to verify the transfer if the token is unpaused, this isn't a mint txn, and there exists a lockup for this user
-        if (!paused && _from != address(0) && lockUps[_from].length != 0) {
+        if (!paused && _from != address(0) && lockUps[_from].length != 0) {  
             // check if this transfer is valid
-            return _checkIfValidTransfer(_from, _amount, _isTransfer);
+            return _checkIfValidTransfer(_from, _amount);
         }
         return Result.NA;
     }
 
     /**
      * @notice Lets the admin create a volume restriction lockup for a given address.
      * @param _userAddress Address of the user whose tokens should be locked up
+     * @param _lockupAmount Amount of tokens that need to lock.
+     * @param _startTime When this lockup starts (seconds)
      * @param _lockUpPeriodSeconds Total period of lockup (seconds)
      * @param _releaseFrequencySeconds How often to release a tranche of tokens (seconds)
-     * @param _startTime When this lockup starts (seconds)
-     * @param _totalAmount Total amount of locked up tokens
      */
     function addLockUp(
```

Comment:
This can be broken into an internal function so that when calling `addLockUpMulti`, `withPerm(ADMIN)` is not checked again and again.